packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGridList.tsx

```diff
@@ -60,13 +46,15 @@ type Props = Partial<BpkCalendarGridProps> & {
   focusedDate?: Date | null;
   selectionConfiguration?: SelectionConfiguration;
   className?: string | null;
+  monthHeight?: number;
 };
 
 const BpkScrollableCalendarGridList = (props: Props) => {
   const {
     className = null,
     focusedDate = null,
     minDate,
+    monthHeight = 8.125,
```

Comment:
`8.125` is the default value that Backpack is currently using, we will stick with this if a prop is not passed to ensure nothing breaks.